.github/workflows/test.yml

```diff
@@ -33,7 +33,10 @@ jobs:
       - name: Install gotestsum
         run: go install gotest.tools/gotestsum@v1.11.0
 
-      - name: Test
+      - name: Run go vet
+        run: go vet ./...
```

Comment:
can we move this to another job

otherwise it runs for all golang versions and on different systems because of the matrix